src/cli.yaml

```diff
@@ -82,6 +82,28 @@ subcommands:
             about: Sets the output file to use
             required: true
             index: 2
+  - update:
+      about: >
+          Attempts to update an exsiting Valheim server installation. By
+          default this involves checking for an update, if an update is
+          available the server will shut down, updated, and brought back online
```

Comment:
will be shut

Reply:
Fixed in e0d5351b72fdb57865783850e2fc6afa27a433b1